release-notes/10.0/preview/preview3/api-diff/Microsoft.WindowsDesktop.App/10.0-preview3_System.Windows.Forms.md

```diff
@@ -0,0 +1,24 @@
+# System.Windows.Forms
+
+```diff
+  namespace System.Windows.Forms
+  {
+      public class DataObject : System.Windows.Forms.ITypedDataObject, System.Windows.Forms.IDataObject
+      {
+-         public void SetDataAsJson<T>(string format, bool autoConvert, T data);
+      }
+      public class ErrorProvider
+      {
+-         protected override void Dispose(bool disposing);
+-         public int BlinkRate { get; set; }
++         public int? BlinkRate { get; set; }
+-         public System.Windows.Forms.ErrorBlinkStyle BlinkStyle { get; set; }
++         public System.Windows.Forms.ErrorBlinkStyle? BlinkStyle { get; set; }
+-         public bool HasErrors { get; }
++         public bool? HasErrors { get; }
+-         public virtual bool RightToLeft { get; set; }
++         public virtual bool? RightToLeft { get; set; }
++         protected override void Dispose(bool? disposing);
```

Comment:
all those `?` doesn't feel right  
https://github.com/dotnet/winforms/blame/release/10.0-preview3/src/System.Windows.Forms/System/Windows/Forms/ErrorProvider/ErrorProvider.cs#L114
especially `Dispose(bool? disposing)`

Reply:
@JeremyKuhne @Tanya-Solyanik can you please confirm the nullability changes?

Reply:
@carlossanlop Yeah, that's wrong. The code doesn't have that, so I'm not sure why this showed up.

Reply:
The SetDataAsJson overload removal is right though.